.github/workflows/README.md

```diff
@@ -0,0 +1,104 @@
+# Stencil Continuous Integration (CI)
+
+Continuous integration (CI) is an important aspect of any project, and is used to verify and validate the changes to the
+codebase work as intended, do not introduce regressions (bugs), and adhere to coding standards (e.g. formatting rules).
```

Comment:
```suggestion
codebase work as intended, to avoid introducing regressions (bugs), and to adhere to coding standards (e.g. formatting rules).
```

Reply:
👍 fixed in https://github.com/ionic-team/stencil/pull/3332/commits/8f85fe5d6465af68c7989ae9d4ade23d80beb2df